configs/basecall.config

```diff
@@ -0,0 +1,38 @@
+/************************************************
+| CONFIGURATION FILE FOR NAO BASECALL WORKFLOW |
+************************************************/
+
+params {
+    mode = "basecall"
+    debug = true
+
+    // Directories
+    base_dir = "s3://nao-mgs-simon/NAO-ONT-20240905-DCS_RNA2_Test"
+
+    // Run parameters
+    // TODO: Automate getting run name from base_dir
+    nanopore_run = "NAO-ONT-20240905-DCS_RNA2_Test"
+
+    // TODO: Add optional paramter for ONT kit
+
+    // Files
+    pod5_dir = "${base_dir}/pod5_small/"
```

Comment:
Do these need to be in the config file? If they'll always be the same except for the base_dir you can hardcode them in the pipeline and save the user some thinking.